framework/core/js/src/common/Translator.tsx

```diff
@@ -1,19 +1,27 @@
+import type { Dayjs } from 'dayjs';
 import { RichMessageFormatter, mithrilRichHandler, NestedStringArray } from '@askvortsov/rich-icu-message-formatter';
 import { pluralTypeHandler, selectTypeHandler } from '@ultraq/icu-message-formatter';
 import username from './helpers/username';
 import User from './models/User';
 import extract from './utils/extract';
 import extractText from './utils/extractText';
+import ItemList from './utils/ItemList';
 
 type Translations = Record<string, string>;
 type TranslatorParameters = Record<string, unknown>;
+type DateTimeFormatCallback = (format?: string) => string | void;
```

Comment:
I don't think `void` is a good usage, maybe `string | false` is better. I am afraid of a branch with no return.

just advice, no force.

Reply:
Why would you afraid of it? It's what `override` accepts as callback's return value.